kyo-core/shared/src/main/scala/kyo/Fiber.scala

```diff
@@ -17,14 +17,14 @@ import scala.util.NotGiven
 import scala.util.control.NonFatal
 import scala.util.control.NoStackTrace
 
-opaque type Fiber[E, A] = IOPromise[E, A]
+opaque type Fiber[+E, +A] = IOPromise[E, A]
```

Comment:
I hadn't added variance in earlier versions because of compiler limitations but never revisited it.